src/scmdata/time.py

```diff
@@ -173,30 +177,33 @@ def weekdays(self) -> np.ndarray:
 
 
 class TimeseriesConverter:
+    """
+    Interpolator used to convert data between different time bases
+
+    This is a modified version originally in :mod:`openscm.time.TimeseriesConverter`. The integral preserving interpolation was
+    removed as it is outside the scope of this package.
+
+    Parameters
+    ----------
+    source_time_points: np.ndarray
+        Source timeseries time points
+    target_time_points: np.ndarray
+        Target timeseries time points
+    interpolation_type: str
```

Comment:
```suggestion
    interpolation_type : {"linear"}
```

If we want to follow numpy docstrings (https://numpydoc.readthedocs.io/en/latest/format.html, search for `{'C'`